config/locales/en.yml

```diff
@@ -1015,6 +1015,12 @@ en:
 
 
     member_mailer:
+      app_no_uploaded_files:
+        subject: Please upload qualification files to your Sveriges Hundföretagare application
+        message_text: "This is a reminder that you need up load files for your SHF application....  why they're needed..."
```

Comment:
This is a reminder that you need up load files for your SHF application so that it can be reviewed.